export.py

```diff
@@ -2350,6 +2352,9 @@ def export_render_settings(ri, rpass, scene, preview=False):
     ri.PixelFilter(rm.pixelfilter, rm.pixelfilter_x, rm.pixelfilter_y)
     ri.ShadingRate(rm.shadingrate)
     ri.Attribute("trace", depths)
+    if rm.use_separate_path_depths:
+        ri.Option("trace", {'string depthmode': 'separate'})
```

Comment:
Sorry I mean separate path depths.  
 
More to the point though I guess this feels like advanced option no?